Formula/cava.rb

```diff
@@ -0,0 +1,60 @@
+class Cava < Formula
+  desc "Console-based Audio Visualizer for ALSA"
+  homepage "https://github.com/karlstav/cava"
+  url "https://github.com/karlstav/cava/archive/0.7.4.tar.gz"
+  sha256 "fefd3cc04d41b03ca416630cafadbfda6c75e2ca0869da1f03963dcb13e1ecb7"
+  license "MIT"
+  head "https://github.com/karlstav/cava.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool"  => :build
+  depends_on "fftw"
+  depends_on "iniparser"
+  depends_on "portaudio"
+
+  uses_from_macos "ncurses"
+
+  def install
+    # change ncursesw to ncurses
+    inreplace "configure.ac", "ncursesw", "ncurses"
```

Comment:
If we use Homebrew `ncurses` we won't need this.

Reply:
I have to problem using homebrew ncurses. But how do I use this?
When I add a "depends_on ncurses" the testbot tells me to use "uses_from_macos"...